core/src/wallet/pool/WalletPool.hpp

```diff
@@ -152,8 +156,9 @@ namespace ledger {
                 const std::shared_ptr<api::RandomNumberGenerator> &rng,
                 const std::shared_ptr<api::DatabaseBackend> &backend,
                 const std::shared_ptr<api::DynamicObject> &configuration,
-                const std::shared_ptr<api::PreferencesBackend> &externalPreferencesBackend,
-                const std::shared_ptr<api::PreferencesBackend> &internalPreferencesBackend);
+                std::shared_ptr<api::PreferencesBackend> externalPreferencesBackend,
```

Comment:
Why not `const`?

Reply:
I use `std::move` instead of `const &` in the implementation (clang tidy proposal)